### PR TITLE
minor fixes for dashboard page. does not include changes for add file…

### PR DESCRIPTION
**Pull request by ogwuazors**
…s buttons/cart

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/pages/dashboardTab/store/dashboardReducer.js [hunk 1]
```diff
@@ -584,8 +584,8 @@ export async function fetchAllFileIDsForSelectAll(fileCount = 100000) {
   /* const SELECT_ALL_QUERY = getState().currentActiveTab === tabIndex[1].title
     ? GET_ALL_FILEIDS_FROM_FILESTAB_FOR_ADD_ALL_CART
     : getState().currentActiveTab === tabIndex[1].title
-      ? GET_ALL_FILEIDS_FROM_SAMPLETAB_FOR_ADD_ALL_CART
-      : GET_ALL_FILEIDS_FROM_CASESTAB_FOR_ADD_ALL_CART; */
+      ? GET_ALL_FILEIDS_FROM_SAMPLETAB_FOR_ADD_ALL_CAR0T
```
Comment (iksheth) [inline src/pages/dashboardTab/store/dashboardReducer.js]: I think there is spelling error in "CAR**0**T". Can you have a look @ogwuazors ?

**Review by iksheth — CHANGES_REQUESTED** [review]

**Comment by ogwuazors** [comment]
made some changes

File: src/pages/dashboardTab/store/dashboardReducer.js (hunk 1 above)
Comment (ogwuazors) [reply src/pages/dashboardTab/store/dashboardReducer.js]: I;ve made the change in a new commit

**Review by ogwuazors — COMMENTED** [review]
the changes should be complete

**Review by iksheth — APPROVED** [review]

**Pull request merged**